index.html

```diff
@@ -6,22 +6,40 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css">
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css">
-    <link rel="stylesheet" href="css/style.css">
-    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <title>Portofilio: mobile version</title>
 </head>
 
-<body>
+<body>  
 
-<section>
+   <container id='wraper'>
 
-</section>
-<section>
+     <div class="app">
+      <nav class= 'navbar'>
+       <h1 class='name'>Dejan Vujovic</h1>
```

Comment:
- [ ] Hi, you've done well to include the logo here. as part of the requirements please kindly ensure that the Logo has an `a` tag to link to the homepage 👍. something like this would be really nice 
```suggestion
       <a href="/"><h1 class='name'>Dejan Vujovic</h1></a>
```